app/src/main/java/com/mapbox/maps/testapp/examples/AddOneMarkerSymbolActivity.kt

```diff
@@ -0,0 +1,76 @@
+package com.mapbox.maps.testapp.examples
+
+import android.graphics.BitmapFactory
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import com.mapbox.geojson.Point
+import com.mapbox.maps.CameraOptions
+import com.mapbox.maps.Style
+import com.mapbox.maps.extension.style.image.image
+import com.mapbox.maps.extension.style.layers.generated.symbolLayer
+import com.mapbox.maps.extension.style.layers.properties.generated.IconAnchor
+import com.mapbox.maps.extension.style.sources.generated.geoJsonSource
+import com.mapbox.maps.extension.style.style
+import com.mapbox.maps.testapp.R
+import kotlinx.android.synthetic.main.activity_add_marker_symbol.*
+
+/**
+ * Example showing how to add a marker on map with symbol layer
+ */
+class AddOneMarkerSymbolActivity : AppCompatActivity() {
+
+  override fun onCreate(savedInstanceState: Bundle?) {
+    super.onCreate(savedInstanceState)
+    setContentView(R.layout.activity_add_marker_symbol)
```

Comment:
nit: if we're still doing `jumpTo` below - we could use `activity_simple_map.xml` here.